docs/how-to/version-artifacts.rst

```diff
@@ -115,3 +115,62 @@ If you log the artifact to an experiment and call :py:meth:`lazyscribe.experimen
 calling :py:meth:`lazyscribe.project.Project.save`, it will behave exactly as if you called
 :py:meth:`lazyscribe.repository.Repository.log_artifact` -- *you* will be responsible for calling
 :py:meth:`lazyscribe.repository.Repository.save`.
+
+Create associated groups of artifact-versions
+---------------------------------------------
+
+.. important::
+
+    New in 2.0.0.
+
+While versioning individual artifacts is useful, oftentimes we want to create groups of related assets. These assets
+have implicit compatibility, allowing users to time-travel through an entire deployment. We have implemented this
+type of functionality through *releases*.
+
+All we need is a repository:
+
+.. code-block:: python
+
+    from lazyscribe import Repository
+    from lazyscribe import release as lzr
+
+    repository = Repository(..., mode="r")
+    release = lzr.create_release(repository, "v0.1.0")
+
+The output :py:class:`lazyscribe.release.Release` object contains 3 attributes:
```

Comment:
In the other code, we say
* "a `Class`" for an instance of `Class`,
* "a `Class` instance" for an instance of `Class`,
* " a Python Object" for a `dict`/etc. object.

And IMHO "a `Class` instance" is a better pattern than "a `Class` object".

So, here:
```suggestion
The output :py:class:`lazyscribe.release.Release` instance contains 3 attributes:
```